src/Resources/views/Shop/GiftCard/pdf.html.twig

```diff
@@ -9,16 +9,7 @@
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/normalize.css@8.0.1/normalize.min.css">
         <style>
-            body {
-                font-family: sans-serif;
-                font-size: 3rem;
-                background: transparent url("{{ path }}") no-repeat left bottom;
-                background-size: 100px;
-            }
-
-            img {
-                display: block;
-            }
+            {{ configuration.pdfRenderingCss|default('')|ssgc_replace_css_option(_context)|raw }}
```

Comment:
How do we use some kind of default value if the user hasn't added any CSS?

We are also still missing the output of the gift card HTML on the gift card configuration page so that users easily can see the selectors they can use in the CSS, right?

Reply:
Right, there is no default value as of now.
Neither display of the HTML.
I'll add that.